README.md

```diff
@@ -263,6 +263,15 @@ $ sudo dnf copr enable carlwgeorge/ripgrep
 $ sudo dnf install ripgrep
 ```
 
+If you're an **openSUSE Leap 15.0** user, you can install ripgrep from the
+[utilities repo](https://build.opensuse.org/project/show/utilities/ripgrep):
+
+```
+$ sudo zypper ar https://download.opensuse.org/repositories/utilities/openSUSE_Leap_15.0/utilities.repo
```

Comment:
Does this need to be updated for every new major release of openSUSE?

Reply:
Yes, this depends on the release